CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Tracking/Utilities/IOSSystem.swift

```diff
@@ -9,6 +9,7 @@ import UIKit
 internal class IOSSystem {
     private let device = UIDevice.current
     let manufacturer: String = "Apple"
+    let platform: String = "iOS"
```

Comment:
[Clarification] should we handle iPad OS here?

Reply:
Done. Thank you!